### ISSUE TITLE
[jshow] - colour contrast in side box

### ISSUE THREAD

**Issue by commscath** [joomla/joomla-websites#138]
https://joomla-showcase.octopoos-dev.com/browse-sites/www-sunliner-com-au.html

Doesn't meet accessibility requirements for colour contrast:

Foreground: #7a8d8e
Background: #ebf0f1

Thanks for your contribution to the project!

Rene

**Comment by MAT978** [comment]
Hello,
The Joomla! Showcase Directory is still under development.
thanks

**Comment by pbwebdev** [comment]
This is from #joomladayau. Teaching group of people how to contribute to Joomla by reporting issues and test for accessibility.

Showcase team will pick them up from here and make the changes on the new dev site.

Thanks @MAT978

**Comment by commscath** [comment]
Excellent workshop @pbwebdev

**Comment by MAT978** [comment]
@pbwebdev 
:+1:  following you, made me realise once again (in fact I didn't need it again ;) ) how I would have loved sharing with you all.
Thanks and have fun!

**Comment by oliviernolbert** [comment]
Contrast enhanced. Thanks.

**Comment by MATsxm** [comment]
Thanks
closed - re open if needed
